libraries/AP_Logger/LoggerMessageWriter.cpp

```diff
@@ -187,6 +187,8 @@ void LoggerMessageWriter_WriteSysInfo::reset()
 
 void LoggerMessageWriter_WriteSysInfo::process() {
     const AP_FWVersion &fwver = AP::fwversion();
+    const char *prot = hal.rcin->protocol();
+    const AP_BattMonitor &battery = AP::battery();
```

Comment:
Use braces to give these a tighter scope for the relevant `case` clauses.

Reply:
Fixed